portfolio.html

```diff
@@ -135,5 +152,26 @@ <h1 class="AboutHead">About me</h1>
       </li>
     </ul>
   </section>
+  <section class="formPage">
+    <h1 class="about-me">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form action="https://formspree.io/f/xvodlvrj" method="POST">
+      <input type="text" name="name" id="name" placeholder="Full name" class="input-name"> <br>
+      <input type="email" id="email" placeholder="Email address" class="input-email"><br>
+      <textarea name="text">Enter text here</textarea>
```

Comment:
- [ ] Please make the all the input fields **required** as the requirement says.

![llll](https://user-images.githubusercontent.com/61562626/134188278-a667f323-3728-4aa7-a184-16dd1736c32d.jpeg)